src/components/SealCred.tsx

```diff
@@ -78,24 +72,23 @@ export default function () {
   return (
     <div className={blockWrapper}>
       <div className={card}>
-        <div className={cardHeader}>
-          <SealCredLogo />
-          <PrimaryAccentText color="text-accent">SealCred</PrimaryAccentText>
-        </div>
-        <div className={cardBody}>
-          <HeaderText>
-            Privacy-preserving social protocol built with ZK soul-bound NFTs
-          </HeaderText>
-          <BodyText>
-            <AccentText color="text-accent">SealCred</AccentText> allows you to
-            experience the world pseudonymously with zero knowledge verified
-            NFTs. This means you can prove credentials, ownership, or facts
-            without them tracing back to you.
-          </BodyText>
-          <Button outlined url="https://sealcred.xyz/">
-            Get started
-          </Button>
-        </div>
+        <ProjectBlock
+          logo={<SealCredLogo />}
+          titleColor="text-accent"
+          titleText="SealCred"
+          subtitle="Privacy-preserving social protocol built with ZK soul-bound NFTs"
+          description={
+            <>
+              <AccentText color="text-accent">SealCred</AccentText> allows you
+              to experience the world pseudonymously with zero knowledge
+              verified NFTs. This means you can prove credentials, ownership, or
+              facts without them tracing back to you.
+            </>
```

Comment:
Should we put it into `BodyText`? What do you think about moving it into a separate component?

Reply:
BodyText is not necessary, I guess:
![image](https://user-images.githubusercontent.com/49658988/192776587-e524870b-873d-4b0a-aefe-b3a0198136a6.png)
![image](https://user-images.githubusercontent.com/49658988/192776614-704866ec-70ab-4426-a408-ba2037572459.png)

Reply:
![image](https://user-images.githubusercontent.com/49658988/192776697-19345ecf-d33b-4035-ad14-ff59e1d45606.png)
It's wrapped inside the component itself

Reply:
I unwrapped other components as well, thanks for pointing this one :)